tracer/src/Datadog.Trace/IAST/Settings/IastSettings.cs

```diff
@@ -0,0 +1,35 @@
+// <copyright file="IastSettings.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using Datadog.Trace.AppSec;
+using Datadog.Trace.Configuration;
+
+namespace Datadog.Trace.Iast.Settings
+{
+    internal class IastSettings
+    {
+        public static readonly string InsecureHashingAlgorithmsDefault = "HMACMD5,MD5,HMACSHA1,SHA1";
+
+        public IastSettings(IConfigurationSource source)
+        {
+            Enabled = source?.GetBool(ConfigurationKeys.Iast.Enabled) ?? false;
+            InsecureHashingAlgorithms = source?.GetString(ConfigurationKeys.Iast.WeakHashAlgorithms) ?? InsecureHashingAlgorithmsDefault;
+            InsecureHashingAlgorithmsArray = InsecureHashingAlgorithms.Split(new char[] { ',' }, System.StringSplitOptions.RemoveEmptyEntries);
```

Comment:
We could update the `Enabled` flag when new types are added too? It's not a big deal though, it would be strange for them to enable IAST but then remove all the algorithms!

Reply:
Ok, makes sense. I have updated the code.